frontend/lib/shopify-storefront-sdk/generated/sdk.ts

```diff
@@ -6385,6 +6387,10 @@ export type FindProductQuery = {
                        __typename?: 'Metafield';
                        value: string;
                     } | null;
+                    enabled?: {
```

Comment:
I think this should be using the enabled2 metafield? I'm surprised this would work with enabled since there's no definition for that.

Reply:
Oh I just saw lower in the file this is using enabled2 but saving it as enabled.

Reply:
The graphQL query below (this is just the types) renamed the `enabled2` metafield in the query:
![image](https://user-images.githubusercontent.com/26855/198105220-0703f811-7d94-4739-acd4-70d2eedeb742.png)